docs/modeling-your-data/modeling-your-data-with-dbt/index.md

```diff
@@ -5,9 +5,14 @@ sidebar_position: 10
 
 ```mdx-code-block
 import ThemedImage from '@theme/ThemedImage';
+import { AllAccelerators } from "@site/src/components/AceleratorAdmonitions";
```

Comment:
sp => `AcceleratorAdmonitions` or is the original component misnamed?

Reply:
Apparently that has just always been misspelt, I'll fix that